app/controllers/pageflow/application_controller.rb

```diff
@@ -21,8 +21,14 @@ class ApplicationController < ActionController::Base
 
     rescue_from ActiveRecord::RecordNotFound do
       respond_to do |format|
-        format.html { render :file => Rails.root.join('public/404.html'), :status => :not_found }
-        format.any(:json, :css) { head :not_found }
+        format.html do
+          begin
+            render file: Rails.public_path.join('pageflow', 'error_pages', '404.html'), status: :not_found
```

Comment:
As far as testing response content, that has little additional value to me over the status code. Also it makes the test brittle in case someone wants to put in new text in this file.